src/util/sss_iobuf.h

```diff
@@ -102,6 +112,11 @@ size_t sss_iobuf_get_size(struct sss_iobuf *iobuf);
  */
 uint8_t *sss_iobuf_get_data(struct sss_iobuf *iobuf);
 
+/*
+ * @brief Returns whether the data is secured
+ */
+bool sss_iobuf_get_secure(struct sss_iobuf *iobuf);
```

Comment:
`const struct sss_iobuf...`

Reply:
The rest of the `sss_iobuf_get_*()` functions don't use the `const`. I could set them all to `const` in a separate commit.